CHANGES

```diff
@@ -1,3 +1,14 @@
+Release 2.3.0 (January 16, 2020)
+===============================
+
+* Fix formatting for pyupgrade
+* Fix error with get_objects_for_user on PostgreSQL
+* Modify 'assign_perm' shortcut to treat lists as a queryset
+* Updated links to Django documentation and django-authority repository
+* Removed reference to "easy_install" in the documentation
+* Run migrations check only if required version of django is installed
+* Fix compatibility with UUIDField primary keys
+
```

Comment:
Can you update that PR to reflect latest changes?

Reply:
added "Drop support for Django 2.1"